app/translations/en/LC_MESSAGES/messages.po

```diff
@@ -634,3 +643,13 @@ msgstr ""
 #~ msgid "Enter a valid access code"
 #~ msgstr ""
 
+#~ msgid ""
+#~ "<p>To collect information about how you"
+#~ " use this website, we "
+#~ "use%(open_cookies_link)s cookies.%(close_link)s</p>\n"
+#~ "                        <p>This information helps"
+#~ " us improve our services and make "
+#~ "sure that the website works as "
+#~ "well as possible.</p>"
+#~ msgstr ""
```

Comment:
Can this be deleted?

Reply:
Yeah, this is just what Babel does with the old content when the content on the page changes.